setup.sh

```diff
@@ -356,6 +360,11 @@ if [ $download == "yes" ]; then
     export PATH=~/.wks/bin:$PATH
 fi
 
+# On macOS, we only support the docker backend.
+if [ $(goos) == "darwin" ]; then
+    set_config_backend docker
+fi
+
```

Comment:
When testing, this leaves a file `config.yaml-e` in the repo and test script terminates before creating the cluster.

Reply:
Gah! I forgot `-i` was a GNU sed thing!

Reply:
Reworked that part, hopefully to a working state.